src/main/java/bizbook/logic/commands/DeleteTagCommand.java

```diff
@@ -0,0 +1,102 @@
+package bizbook.logic.commands;
+
+import static bizbook.commons.util.CollectionUtil.requireAllNonNull;
+import static bizbook.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import bizbook.commons.core.index.Index;
+import bizbook.logic.Messages;
+import bizbook.logic.commands.exceptions.CommandException;
+import bizbook.model.Model;
+import bizbook.model.person.Address;
+import bizbook.model.person.Email;
+import bizbook.model.person.Name;
+import bizbook.model.person.Note;
+import bizbook.model.person.Person;
+import bizbook.model.person.Phone;
+import bizbook.model.tag.Tag;
+
+/**
+ * Delete a tag from an existing person in the address book.
+ */
+public class DeleteTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "deletetag";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the tag from a specfic person.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_TAG + "TAG]...\n"
```

Comment:
Square brackets in the syntax imply that it is optional but `TAG` is not an optional parameter.